kernel/src/log_segment.rs

```diff
@@ -288,9 +288,13 @@ fn list_log_files(
 
     Ok(fs_client
         .list_from(&start_from)?
-        .map(|meta| ParsedLogPath::try_from(meta?))
-        // TODO this filters out .crc files etc which start with "." - how do we want to use these kind of files?
+        .map(|meta| meta.map(|m| ParsedLogPath::try_from(m).ok().flatten()))
```

Comment:
(as @nicklan pointed out - I'll just post here and let him follow up with a review)

I think this will ignore parsing errors and we likely want to propagate the error from this `try_from`.